csharp/Platform.IO/StreamExtensions.cs

```diff
@@ -8,6 +8,23 @@ namespace Platform.IO
 {
     public static class StreamExtensions
     {
+        /// <summary>
+        /// <para>Writes a sequence that represents the <paramref name="value"/> to the <paramref name="stream"/> and moves the current position of the <paramref name="stream"/> by the number of written bytes.</para>
+        /// <para>Записывает последовательность байт представляющую <paramref name="value"/> в поток <paramref name="stream"/> и перемещает текущую позицию в <paramref name="stream"/> вперед
+        на число записанных байт.</para>
```

Comment:
Here the line break symbol breaks the comment structure and will lead to compile error.